core/eolearn/core/eodata.py

```diff
@@ -499,7 +499,8 @@ def __deepcopy__(self, memo: Optional[dict] = None, features: FeaturesSpecificat
     def copy(self, features: FeaturesSpecification = ..., deep: bool = False) -> EOPatch:
         """Get a copy of the current `EOPatch`.
 
-        :param features: Features to be copied into a new `EOPatch`. By default, all features will be copied.
+        :param features: Features to be copied into a new `EOPatch`. By default, all features will be copied. Note that
+            `BBOX` is always copied.
```

Comment:
When we reach the point where stop referring to the BBox as one of the features, this should also be removed.  Perhaps it's safer to remove the additional comment due to this?

Reply:
It should be noted somewhere. I can move it to the general body of the docstring

Reply:
I tried moving it into the body so that it was "Get a copy of the current `EOPatch`. The `bbox` is always copied.", but i'm not sure it's much better

Reply:
I mean't it more in the direction if it even makes sense in the first place to add this comment, since this would hold by default as a consequence of the assumption that we cannot have an EOPatch without a bbox. Or am I again looking too far ahead?

Reply:
Nevermind, I see that this comment is already in line with the future changes. Resolving.